source/ada/lsp-ada_handlers.adb

```diff
@@ -1351,6 +1353,61 @@ package body LSP.Ada_Handlers is
 
          --  Refactoring Code Actions
 
+         --  Extract Subprogram
+         declare
```

Comment:
this begin block is big enough: worth extracting to a local subprogram!! (using this feature!)